File: uri/1180/1180.cpp
#include <cstdio>
#include <iostream>

using namespace std;

int main(){
    int c, n, m, im = 0, i = 0;
    cin >> c;
    cin >> n;
    m = n;
    while(c--){
        i++;
        cin >> n;
        if (n < m){
            im = i;
            m = n;
        }
    }
    printf("Menor valor: %d\n", m);
    printf("Posicao: %d\n", im);
    return 0;
}

File: solutions/beecrowd/1180/1180.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

n = int(input())
xs = list(map(int, input().split()))

min_x = min(xs)
index = xs.index(min_x)

output(f'Menor valor: {min_x}\n')
output(f'Posicao: {index}\n')
